spec/statement.md

```diff
@@ -0,0 +1,131 @@
+# Statement
+
+## Table of Contents
+
+* [Definitions](#definitions)
+  * [Current indentation level](#current-indentation-level)
+  * [Document indentation character](#document-indentation-character)
+* [Syntax](#syntax)
+  * [ScssStatements](#scssstatements)
+  * [IndentedStatements](#indentedstatements)
+  * [Statements](#statements)
+  * [Block](#block)
+  * [Whitespace](#whitespace)
+  * [Indentation](#indentation)
+
+## Definitions
+
+### Current indentation level
+
+The `current indentation level` is the count of the [`documentation indentation
+character`] between a new line and any non-whitespace characters of the last
+consumed line.
```

Comment:
This should explicitly exclude lines that are only whitespace from consideration.